.gitignore

```diff
@@ -396,6 +396,3 @@ FodyWeavers.xsd
 
 # JetBrains Rider
 *.sln.iml
-
-# Skip any generated service code
-servers/**/generated/
```

Comment:
I think we should include those generated code, otherwise the files on github actually cannot build, and it would be convenient if we included them here to share those generated code around to see if anything is not so correct.